composer.json

```diff
@@ -52,8 +52,8 @@
         "bower-asset/jquery-timeentry": "^2.0",
         "bower-asset/caret.js": "0.2.2",
         "npm-asset/at.js": "^1.5.1",
-        "yiisoft/yii2-queue": "^2.0",
-        "yiisoft/yii2-redis": "^2.0",
+        "yiisoft/yii2-queue": "~2.0.0",
+        "yiisoft/yii2-redis": "~2.0.0",
```

Comment:
What is reason for replace >=2.0.0 <3.0.0. to >=2.0.0 <2.1.0. ?

Reply:
@githubjeka Sometimes there are breaking changes between 2.0 and 2.1 (e.g. https://github.com/yiisoft/yii2-authclient/blob/master/UPGRADE.md)